scipy/stats/_multivariate.py

```diff
@@ -280,7 +281,7 @@ class multivariate_normal_gen(object):
         Probability density function.
     logpdf(x, mean=None, cov=1, allow_singular=False)
         Log of the probability density function.
-    rvs(mean=None, cov=1, allow_singular=False, size=1)
+    rvs(mean=None, cov=1, size=1, random_state=None)
```

Comment:
@ev-br I just removed the `allow_singular` keyword because it does not seem to be used anymore.